CMakeLists.txt

```diff
@@ -7,9 +7,6 @@ endif()
 if(POLICY CMP0063)
   cmake_policy(SET CMP0063 NEW)
 endif()
-if(POLICY CMP0059)
-  cmake_policy(SET CMP0059 OLD) # we use DEFINITIONS as a built-in directory property
```

Comment:
According to this comment, this is because of

    resources/sugarcrm/CMakeLists.txt:3:add_definitions(-DTRANSLATION_DOMAIN=\"akonadi_sugarcrm_resource\")

isn't it?

Reply:
Hmm, that doesn't really explain it, AFAICS the policy is about extracting that property again, which we're not doing.
And the definition is definitely still there after your commit.

This is curious indeed.